src/test/java/org/ilgcc/app/journeys/GccFlowJourneyTest.java

```diff
@@ -231,9 +236,16 @@ void fullGccFlow() throws IOException {
     testPage.clickContinue();
     //activities-employer-address
     assertThat(testPage.getTitle()).isEqualTo("Activities Employer Address");
-    testPage.enter("employerPhoneNumber", "333333333");
+    testPage.enter("employerPhoneNumber", "3333333");
+    testPage.clickContinue();
+    //activities-employer-address page should have an error for invalid phone number
+    assertThat(testPage.getFirstInputError()).isEqualTo("Make sure the phone number you entered includes 9 digits.");
+    testPage.enter("employerPhoneNumber", "3333333333");
     testPage.enter("employerCity", "Chicago");
     testPage.enter("employerStreetAddress", "123 Test Me");
+    testPage.enter("employerZipCode", "6042");
+    testPage.clickContinue();
+    assertThat(testPage.getFirstInputError()).isEqualTo("Make sure the zip code you entered follows the right format.");//
```

Comment:
I think we can combine the error checks
```suggestion
    testPage.enter("employerPhoneNumber", "3333333");
    testPage.enter("employerZipCode", "6042");
    testPage.clickContinue();
    //activities-employer-address page errors
    assertThat(testPage.hasErrorText(("Make sure the phone number you entered includes 9 digits.")).isTrue();
        assertThat(testPage.hasErrorText("Make sure the zip code you entered follows the right format.")).isTrue();
    testPage.enter("employerPhoneNumber", "3333333333");
    testPage.enter("employerCity", "Chicago");
    testPage.enter("employerStreetAddress", "123 Test Me");
```